src/response.rs

```diff
@@ -147,30 +147,6 @@ impl Response {
             .collect()
     }
 
-    /// Whether the response status is: 200 <= status <= 299
-    pub fn ok(&self) -> bool {
-        self.status >= 200 && self.status <= 299
-    }
-
-    pub fn redirect(&self) -> bool {
-        self.status >= 300 && self.status <= 399
-    }
-
-    /// Whether the response status is: 400 <= status <= 499
-    pub fn client_error(&self) -> bool {
-        self.status >= 400 && self.status <= 499
-    }
-
-    /// Whether the response status is: 500 <= status <= 599
-    pub fn server_error(&self) -> bool {
-        self.status >= 500 && self.status <= 599
-    }
-
-    /// Whether the response status is: 400 <= status <= 599
-    pub fn error(&self) -> bool {
-        self.client_error() || self.server_error()
-    }
-
```

Comment:
Why did these go?

Reply:
As @jyn514 pointed out at https://github.com/algesten/ureq/issues/128#issuecomment-732470836, it's confusing to have e.g. `.client_error()` on Response, because the main way you get a Response is via `Ok(response)`, in which case you know ``client_error()` will always be false.

The other way you can get a Response is via `err.into_response()`, but if you are calling `.into_response()` you've probably already checked the status code to see whether it's one you're interested in.

The other answer is that I checked all the published crates for users of these and found they're not used at all. People prefer `.status()`. I figure, let's head into 2.0.0 with a clean slate and a narrow API; if we find there's user demand for methods like `.client_error()`, we can always add them back, but we can't delete them if we launch with them.